src/kitovu/sync/syncing.py

```diff
@@ -7,11 +7,12 @@
 
 from kitovu import utils
 from kitovu.sync import syncplugin
+from kitovu.sync.settings import Settings, PluginSettings
 from kitovu.sync.plugin import smb
 
 
 def _find_plugin(pluginname: str) -> syncplugin.AbstractSyncPlugin:
-    """Find an appropriate sync plugin with the given name."""
+    """Find an appropriate sync plugin with the given settings."""
```

Comment:
This seems wrong, no? This method still only cares about the name.

Reply:
I was about to change from passing the plugin name to passing the plugin settings. After I've started the rewrite I deemed it unnecessary and reverted it and forgot about the comment.

But I have changed to passing the plugin settings in the branch feature-config-file-validation to make the validations simpler